packages/components/src/FormTabs/components/Tabs.tsx

```diff
@@ -29,9 +29,9 @@ export interface IProps extends IButtonProps {
 export const Tab = styled(Button)<IProps>`
   --color: ${({ theme, activeColor }) =>
     activeColor ? activeColor : theme.colors.primary};
-  margin-inline: 8px;
   border-radius: 0;
   margin-top: 8px;
+  margin-right: 16px;
```

Comment:
@jpye-finch Not sure if this would affect other places the tabs are used in, like without the Content component? Have we even used it without the Content component?